templates/_machines-architectures.html

```diff
@@ -0,0 +1,49 @@
+<script src="//cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.15/lodash.min.js" type="text/javascript"></script>
+<script type="text/javascript">
+  (function () {
+    window.jQuery
+      .ajax({
+        url: "https://api.balena-cloud.com/v6/device_type?$expand=is_of__cpu_architecture($select=slug)"
+      })
+      .done(function (deviceTypes) {
+
+        var table;
+
+        table = _(deviceTypes.d)
```

Comment:
Any reason we don't declare it inline?
```suggestion
        var table = _(deviceTypes.d)
```